docs/README.md

```diff
@@ -4,3 +4,5 @@
   <small>Like Prometheus, but for logs!</small> </p>
   
 Please see the [Documentation Site](https://grafana.com/docs/loki/latest/) The files in this directory are used to generate it but consequently the links don't work in Github.
+
+To contribute to these docs, refer to the `Documentation` section in [CONTRIBUTING.md](/CONTRIBUTING.md).
```

Comment:
I think this link would go to `https://github.com/CONTRIBUTING.md`, except Github does some magic with absolute path replacement to point to the root of the correct repository.

```suggestion
To contribute to these docs, refer to the `Documentation` section in [CONTRIBUTING.md](../CONTRIBUTING.md).
```